feg/cloud/go/services/feg/feg/main.go

```diff
@@ -35,6 +38,14 @@ func main() {
 
 	builder_protos.RegisterMconfigBuilderServer(srv.GrpcServer, servicers.NewBuilderServicer())
 
+	specPath := config.GetSpecPath(feg_service.ServiceName)
+	specServicer, err := swagger.NewSpecServicerWithPath(specPath)
+	if err != nil {
+		glog.Infof("Error retrieving Swagger Spec of service %s", feg_service.ServiceName)
```

Comment:
once we have the spec servicer fully implemented, will this ever happen?

Reply:
Hopefully not, but there's definitely the chance that any 3rd party who wants to attach their service might accidentally name their Swagger Spec file some incompatible extension which means their file would not make it to the production image.